tool/python/singa/layer.py

```diff
@@ -228,21 +228,20 @@ def __init__(self, size=0, **kwargs):
 
 class Activation(Layer):
 
-    def __init__(self, activation='stanh', topk=1):
+    def __init__(self, activation='stanh'):
         '''
         required
-          activation = (string)
-        optional
-          topk       = (int)  // the number of results
+          activation = (string) // relu, sigmoid, tanh, stanh, softmax.
         '''
 
+        if activation == 'tanh':
```

Comment:
sorry. I think here is still some problem.
currently, ActivationLayer and cudnnActivationLayer support sigmoid, relu and tanh. 
Besides, there is a STanhLayer which supports stanh.
Hence, when users configure the activation to be tanh for cpu or gpu, we can support it.
When users configure the activation to be stanh, we can create the STanhLayer to support it.
Hence, we may not need to warn the users on tanh for cpu. Am I right?